templates/admin/players.html.twig

```diff
@@ -1,5 +1,6 @@
 {% extends '@EasyAdmin/default/layout.html.twig' %}
 {% block content %}
+    <link rel="stylesheet" href="{{ asset('build/tournament.css') }}"/>
```

Comment:
Move this include to html head section, not body. Check the blocks inside '@EasyAdmin/default/layout.html.twig' and extend the proper block for css includes. And also is not clear why do you need css, try using bootstrap as much as possible